plugins/systemair/__init__.py

```diff
@@ -53,6 +55,7 @@ def init_serial_connection(self, serialport, slave_address):
             return False
 
     def _read_modbus(self):
+        self._lockmb.acquire()
```

Comment:
So the  ```init_serial_connection()``` should (re-)initialize the modbus connection after 30sec. In my test cases, the modbus connection seems to be a bit unstable (maybe the chinese usb modbus converter).